mne/io/eeglab/eeglab.py

```diff
@@ -52,8 +52,6 @@ def _check_eeglab_fname(fname, dataname):
             "Old data format .dat detected. Please update your EEGLAB "
             "version and resave the data in .fdt format"
         )
-    elif fmt != ".fdt":
-        raise OSError("Expected .fdt file format. Found %s format" % fmt)
```

Comment:
If `.fdt` is standard, it's perhaps better to change this to a `warn(...)` rather than remove it entirely

Reply:
Yes, `.fdt` is the standard when the data is not included in the `EEG` structure.
However there is a warning already in place a few lines down (line 69) if the `.fdt` file is not found, which already covers 1) there is no `.fdt` file, or 2) the `.fdt` file name is incorrect.

So, I did not add another warning to avoid issuing double warnings for a single issue.